File: java/DataStrucure/src/DynamicCollection/Stack.java
package DynamicCollection;

public class Stack<T> {

  private SingleLinkedList<T> myStack;

  public Stack() {
    myStack = new SingleLinkedList<>();
  }

  public boolean push(T value) {
    return myStack.addToFirst(value);
  }

  public boolean pop() {
    return myStack.delFromFirst();
  }

  public boolean printList() {
    return myStack.printList();
  }

  public int getLength() {
    return myStack.getLength();
  }

}

File: python/stack/stack.py
from ..singleList.singleList import SingleLinkedList

class Stack:
    def __init__(self):
        self.myStack = SingleLinkedList()

    def push(self, value):
        return self.myStack.add_to_first(value)

    def pop(self):
        return self.myStack.del_from_first()

    def print_stack(self):
        return self.myStack.print_list()

    def get_length(self):
        return self.myStack.get_length()
